### PR TITLE
Update tfa support for Gemini

### LINKED ISSUE #4577: update site: Gemini
Gemini now supports Hardware tokens

URL: https://gemini.com/
Category:  Cryptocurrencies

### PR DESCRIPTION
**Pull request by miamimanni**
This is in reference to issue #4577 

----

Before
before

----

After
after

### CODE REVIEW

**Comment by Carlgo11** [comment]
There’s no documentation that confirms that they support any hardware based 2FA.

**Comment by miamimanni** [comment]
Looks like it's mentioned on their security page, see below screenshot. Is it best to update the docs to point to security page instead? Then again the security page doesn't mention SMS / voice. There is a article on medium website that seems to mention SMS / software token / hardware token but is missing Phone Call. What is the best course of action?

image

- Security Page: https://gemini.com/security
- Medium Page: https://medium.com/@marcin_at_gemini/securing-your-gemini-account-with-webauthn-b5f369b8beec

**Review by Carlgo11 — APPROVED** [review]

**Comment by miamimanni** [comment]
Should I change the doc url?

**Comment by Carlgo11** [comment]
No the current one is fine.

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -335,6 +335,7 @@ websites:
       - sms
       - proprietary
       - phone
+      - hardware
```
Comment (Jawshy) [inline _data/cryptocurrencies.yml:336]:
```suggestion
      - hardware
      - u2f
```
FIDO U2F is supported.

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -335,6 +335,7 @@ websites:
...
     doc: https://gemini.com/blog/better-two-factor-authentication/
```
Comment (Jawshy) [inline _data/cryptocurrencies.yml]:
```suggestion
    doc: https://gemini24.zendesk.com/hc/en-us/sections/360001694951
```
Gemini - help centre -  Two-Factor Authentication (2FA)

File: _data/cryptocurrencies.yml [hunk 3]
```diff
@@ -335,6 +335,7 @@ websites:
       - sms
       - proprietary
       - phone
```
Comment (Jawshy) [inline _data/cryptocurrencies.yml]:
```suggestion
```
Although their [2017 blog](https://gemini.com/blog/better-two-factor-authentication) mentions voice, this [2019 Medium](https://medium.com/@marcin_at_gemini/securing-your-gemini-account-with-webauthn-b5f369b8beec) article doesn't mention it and I couldn't find the option for phone call/voice in security settings. When opening a personal account, I was explicitly asked for an SMS-capable phone.

**Review by Jawshy — DISMISSED** [review]
Thanks for your submission. I have left some suggested changes.

**Review by Jawshy — APPROVED** [review]

**Pull request merged**